tap_pendo/streams.py

```diff
@@ -807,7 +829,9 @@ def sync(self, state, start_date=None, key_id=None, parent_last_updated=None):
                     self.last_processed = None
                     break
 
-                removed_records = self.remove_last_timestamp_records(records)
+                # Remove last processed and none replication_value records
+                self.remove_none_replication_value_records(records)
```

Comment:
By using this method here, we are skipping writing these none_replciation records to destination because `EventsBase` has its own `sync` method which overrides the `sync` defined in `Stream` class.
In the `sync` method defined in `Stream` class we are appending these none_replication records [here](https://github.com/singer-io/tap-pendo/blob/63f7728e99c5dc6890731956aed99f0991b1630a/tap_pendo/streams.py#L673) to stream_records variable and I believe we are skipping this step in `sync` method defined in `EventsBase` class. Please verify, I'm not sure about this